src/main/java/com/microsoft/azure/functions/annotation/RetryStrategy.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.functions.annotation;
+
+/**
+ * <p>
+ * Strategy of the Retry annotation. Choose 'EXPONENTIAL' if you use  the exponential back off strategy or 'FIXED'
+ * if you use the fixed delay strategy. For more details, refer to https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-error-pages?tabs=csharp#function-level-configuration .
+ * </p>
+ *
+ * @since 1.0.0
+ */
+public enum RetryStrategy {
+    /**
+     * Defines an exponential backoff retry strategy, where the delay between retries
+     * will get progressively larger, limited by the max/min specified.
+     */
+    EXPONENTIAL,
```

Comment:
Rename to : `ExponentialBackoff ` https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.Abstractions/Description/RetryStrategy.cs

Reply:
Make it `EXPONENTIAL_BACKOFF` to fit the naming convention. Sounds good?

Reply:
Make annotation into two again.